script/Arcueid/Day 7/07_00_ARC07_1B.txt

```diff
@@ -192,22 +220,27 @@ I mean, all of you look so young and delicious looking. I think it's inevitable
 [sha:b63945d3503715f582a634c0328148d998cae224]{
 -- Page 54, Offset 16711.
 -- 分かってます分かってます、例の貧血でしょう？　真面目な志貴クンが火遊びをするワケないものね。
-I know, I know, it's the usual anemia right? An honest boy like you would never go around doing something so dangerous.
+// 火遊 is 'playing with fire', which was previously translated as 'dangerous',
+// but it seems that it can also mean playing with _love_ / flirting about.
+// Do we think Noel is insinuating? Feels plausible given the prior lines.
+I know, I know, it's the usual anemia right?
+ An honest boy like you would never go around playing the field like that.
```

Comment:
I mean... I guess your explanation makes sense. I don't really see how "There's just no way you'd go play with fire like that." or "There's just no way you'd go put yourself in danger" would make a lick of sense in this context.
Maybe ask Gekimon / Tumbling on this one, it's a little out of my pay grade.

Bear in mind though that 2 lines later: 本当はノエル先生の冗談 loses all meaning without 火遊びをする being translated literally because he's talking about the fight with vlov. As such, might be worth just taking the hit on the meaning and dumping "play with fire" or whatever